kalitectl.py

```diff
@@ -219,7 +222,7 @@ def get_pid():
     listen_port = getattr(settings, "CHERRYPY_PORT", LISTEN_PORT)
 
     # Timeout is 1 second, we don't want the status command to be slow
-    conn = httplib.HTTPConnection(LISTEN_ADDRESS, listen_port, timeout=3)
+    conn = httplib.HTTPConnection("127.0.0.1", listen_port, timeout=3)
```

Comment:
Agreed, so we need to grab a valid address (meaning not "0.0.0.0") that we can listen to, regardless of whatever options the user has specified.